src/views/Orders.vue

```diff
@@ -10,32 +10,32 @@
     <ion-content>
       <main>
         <section>
-          <ion-card>
+          <ion-card @click="selectedCard = 'import'">
             <ion-card-header>
-              <ion-card-title>{{ $t("Import") }}</ion-card-title>
+              <ion-card-title :color="highlight(selectedCard,'import')">{{ $t("Import") }}</ion-card-title>
             </ion-card-header>
             <ion-item @click="viewJobConfiguration({ id: 'IMP_NEW_ORDERS', status: getJobStatus(jobEnums['IMP_NEW_ORDERS'])})" detail button>
-              <ion-label class="ion-text-wrap">{{ $t("New orders") }}</ion-label>
+              <ion-label class="ion-text-wrap" :color="highlight(jobEnums['IMP_NEW_ORDERS'],currentJob.systemJobEnumId) ">{{ $t("New orders") }}</ion-label>
               <ion-label slot="end">{{ getTemporalExpression('IMP_NEW_ORDERS') }}</ion-label>
             </ion-item>
             <ion-item @click="viewJobConfiguration({ id: 'APR_ORD', status: getJobStatus(jobEnums['APR_ORD'])})" detail button>
-              <ion-label class="ion-text-wrap">{{ $t("Approve orders") }}</ion-label>
+              <ion-label class="ion-text-wrap" :color="highlight(jobEnums['APR_ORD'],currentJob.systemJobEnumId) ">{{ $t("Approve orders") }}</ion-label>          
```

Comment:
in currentJob.systemJobEnumId

systemJobEnumId should be optional property.
```suggestion
              <ion-label class="ion-text-wrap" :color="highlight(jobEnums['APR_ORD'],currentJob?.systemJobEnumId) ">{{ $t("Approve orders") }}</ion-label>          
```